server/src/linker.js

```diff
@@ -32,3 +32,13 @@ exports.staticLink = function (resource) {
 exports.staticLinkWithHost = function (req, resource) {
   return req.protocol + "://" + req.headers.host + exports.staticLink(resource);
 };
+
+exports.imageLink = function (urlBase, resource) {
+  if (resource.charAt(0) != "/") {
```

Comment:
Because I copied and pasted from code you wrote, otherwise I would have written !== ;-)